src/ixdat/measurements.py

```diff
@@ -333,49 +531,74 @@ def __add__(self, other):
 
 
 #  ------- Now come a few module-level functions for series manipulation ---------
```

Comment:
Is this where we would also have integration functions? (such as in Kernel.sig_area?)

Reply:
Not here. Those would be bound to a class.

But I can't decide whether it should go in `Measurement` or in `ValueSeries`. In principle the latter has all it needs (the value np array and the time np array). Which do you prefer of these two (`meas` is a `Measurement` instance)?
1. `int_M2 = meas.calc_integral("M2", tspan=[100, 200])`
2. `int_M2 = meas["M2"].calc_integral(tspan=[100, 200])`